src/fastapi_poe/base.py

```diff
@@ -269,8 +270,20 @@ def error_event(
             data["error_type"] = error_type
         return ServerSentEvent(data=json.dumps(data), event="error")
 
+    @staticmethod
+    def inline_attachment_event(*, inline_ref: str, description: Optional[str] = None):
+        if description:
+            text = f"![{_markdown_escape(description)}][{inline_ref}]"
+        else:
+            text = f"![{inline_ref}]"
+        return ServerSentEvent(data=json.dumps({"text": text}), event="text")
+
     # Internal handlers
 
+    _attachment_upload_url = (
```

Comment:
It would be good if we could stop hardcoding this URL. A possible alternative approach (which would require a protocol change) is that we send the URL with the QueryRequest from our servers. That way, we can easily change the target URL on our side without relying on people updating their version of fastapi-poe.